### ISSUE TITLE
Firebase admin verifyIdToken crashes

### ISSUE THREAD

**Issue by ghost** [oven-sh/bun#3950]
### What version of Bun is running?

0.7.1

### What platform is your computer?

Darwin 22.5.0 arm64 arm

### What steps can reproduce the bug?

```js
const { initializeApp } = require('firebase-admin/app')
const { credential } = require("firebase-admin")
const { getAuth } = require("firebase-admin/auth")

var serviceAccount =
    process.env.ENVIRONMENT == 'PROD'
        ? require('./prod-admin-sdk.json')
        : require('./stag-admin-sdk.json');

initializeApp({
    credential: credential.cert(serviceAccount),
});

let customUserId = await getAuth().verifyIdToken(access_token);
```

### What is the expected behavior?

The token should be verified and returns a valid object

### What do you see instead?

The code crashes and returns the following error
```25 |  * @param errorInfo - The error information (code and message).
26 |  * @constructor
27 |  */
28 | class FirebaseError extends Error {
29 |     constructor(errorInfo) {
30 |         super(errorInfo.message);
            ^
error: Right hand side of instanceof is not an object
 code: "auth/argument-error"

      at new FirebaseError (/node_modules/firebase-admin/lib/utils/error.js:30:8)
      at new PrefixedFirebaseError (/node_modules/firebase-admin/lib/utils/error.js:65:8)
      at new FirebaseAuthError (/node_modules/firebase-admin/lib/utils/error.js:150:8)
      at mapJwtErrorToAuthError (/node_modules/firebase-admin/lib/auth/token-verifier.js:274:15)
      at /node_modules/firebase-admin/lib/auth/token-verifier.js:246:18
```

**Comment by nutraplatesgm** [comment]
+1

**Comment by O-Q** [comment]
I investigated and this problem is related to the `jsonwebtoken` library and this [issue](https://github.com/oven-sh/bun/issues/2036).

**Comment by chen-rn** [comment]
Are there any methods to get around this while Bun fixes the jsonwebtoken lib?

**Comment by ghost** [comment]
I am using the 'jose' package, which can so the same thing

**Comment by chen-rn** [comment]
> I am using the 'jose' package, which can so the same thing

Not sure if I completely understand. How were you able to replace jsonwebtoken with jose? Did you patch firebase-admin?

**Comment by Electroid** [comment]
Related to:
* #1454 
* #2036

**Comment by Electroid** [comment]
Fixed by #5940
